Source/AliveLibAE/Sound/PsxSpuApi.cpp

```diff
@@ -1346,19 +1347,51 @@ EXPORT void CC SsSetTickMode_4FDC20(s32)
     // Stub
 }
 
-// TODO: Removed 4FDC30
+EXPORT bool CC SsSeqCalledTbyT_4FDC80_sleep_until_due_time_if_before_max_time(SYS_time_point_t current_time, SYS_time_point_t max_time)
```

Comment:
_4FDC80 needs to be at the end of the function as exported function names are parsed at runtime for installing hooks into the game when running as a DLL. This name is super verbose too, I would have just left it as SsSeqCalledTbyT_4FDC80 tbh since the waiting/sleeping is an implementation detail.

Reply:
I meant to avoid breaking existing SsSeqCalledTbyT_4FDC80 ABI with "max_time" new argument.
(because I understood that number-suffixed functions are meant to be interchangeable with original binaries)
Unless I miss something, I thus can simplify the name but not use the original one.